krun/platform.py

```diff
@@ -963,9 +977,12 @@ def _check_isolcpus(self):
 
             break  # found the isolcpus arg
         else:
-            self._advise_isolcpus_arg([], expect_cpus)  # exits
+            v = ""  # didn't find a isolcpus arg
 
-        got_cpus = list(sorted(v.split(",")))
+        if v == "":
+            got_cpus = []
+        else:
+            got_cpus = list(sorted(v.split(",")))
```

Comment:
I was trying to protect against "isolcpus=" in the kernel cmdline, but actually I don't think that is valid.